pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -521,6 +557,33 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	// Allow kops-controller to the master instances from the ELB
+	if b.Cluster.UsesNoneDNS() && b.APILoadBalancerClass() == kops.LoadBalancerClassNetwork {
```

Comment:
Should we prohibit NoneDNS with Classic API loadbalancers?

Reply:
Sure, done.